modules/lib/mail.nix

```diff
@@ -0,0 +1,8 @@
+{ config, lib }:
+{
+  isGmail = account:
```

Comment:
Better add a "gmail" option in https://github.com/rycee/home-manager/blob/9570cedff62a6d08eb13f8c6164b496ff88c834f/modules/accounts/email.nix#L146-L158